### DISCUSSION TITLE
Feedback on the osu!lazer experience

### DISCUSSION THREAD

**Discussion by Fabiano1337** [ppy/osu#24770] [Ideas]
Hi,
Im a rank 14k osu! player and recently tried osu!lazer due to OpenTabletDriver bugging out and osu!lazer having an implimented driver, this is my experience from switching to lazer so far :

+ The game itself runs smooth
+ The setup went great, the first start guide was really usefull to have and a nice introduction
+ Tablet driver works great out of the box

-  The map import took 4 hours for ~44k difficulties  / maybe you could speed up the progress by multithreading or something
-  Tablet driver does not have an option to add a smoothing filter
-  Tablet driver does not have an option to change the sensitivity for the pen tip (mouse click on 30% for example)
-  Random map select (F2) is not instant like in classic osu! (although it is a matter of taste if it's better instant or with a little delay)
-  Skin sound editing is really time intensive as you have to export the skin, replace the sounds, delete the skin from osu!lazer and reimport it

A feature i'd like to see added is DT/NC getting a fixed bpm option so you don't have to work with ratios

**Comment by peppy** [comment]
Thanks for your feedback!

> The map import took 4 hours for ~44k difficulties / maybe you could speed up the progress by multithreading or something

It's actually limited by web requests to get metadata for some beatmaps. We'll fix this in the future, but for what it's worth there's already plenty of threading. You can test how fast things are when logged out (but your beatmaps won't have correct linkage information to online systems, so it's best you don't do this).

> Skin sound editing is really time intensive as you have to export the skin, replace the sounds, delete the skin from osu!lazer and reimport it

Yeah, bit unfortunate but we'll improve this in the future.
